app/assets/javascripts/modules/moj.AgeLabel.js

```diff
@@ -0,0 +1,83 @@
+// Age labels for MOJ
+// Dependancies: moj, jQuery
+
+(function () {
+
+  'use strict';
+
+  var AgeLabel = function($el, options) {
+    this.init($el, options);
+    return this;
+  };
+
+  AgeLabel.prototype = {
+
+    defaults: {
+      threshold: 18,
+      labelUnder: 'Child',
+      labelOver: 'Adult'
+    },
+
+    init: function($el, options) {
+      this.settings = $.extend({}, this.defaults, options);
+      this.cacheEls($el);
+      this.bindEvents();
+      this.updateLabel();
+    },
+
+    cacheEls: function($el) {
+      this.$date = $el.find('.AgeLabel-date');
+      this.$label = $el.find('.AgeLabel-label');
+    },
+
+    bindEvents: function() {
+      this.$date.on('change', 'input', $.proxy(this.updateLabel, this));
+    },
+
+    isValidDate: function() {
+      var year = this.$date.find('.year').val(),
+          month = this.$date.find('.month').val(),
+          day = this.$date.find('.day').val();
+
+      if (year!=='' && month !== '' && day !== '') {
+        return new Date(year, month-1, day);
+      }
+    },
+
+    updateLabel: function() {
+      var type,
+          date = this.isValidDate();
+
+      if (date) {
+        this.$label.hide();
+
+        type = this.getYears(date) >= this.settings.threshold ? 'Over' : 'Under';
+
+        this.$label
+          .show()
+          .removeClass('under over')
+          .addClass(type.toLowerCase())
+          .text(this.settings['label' + type]);
+      }
+    },
+
+    getYears: function(from, to) {
+      var diff;
+      to = to || new Date();
+      diff = to.getTime() - from.getTime();
+      return Math.floor(diff / (1000 * 60 * 60 * 24 * 365.25));
+    }
```

Comment:
I don't feel it's necessary to define the magic numbers in that way. I feel it's purpose and method name already describe enough about what those numbers do. It doesn't take long to work it out.